provisioning/device/src/polling_state_machine.ts

```diff
@@ -32,8 +32,8 @@ export class  PollingStateMachine extends EventEmitter {
               callback(err, body, result);
             }
           },
-          register: (callback, registrationId, authorization, requestBody, forceRegistration) => {
-            this._fsm.transition('sendingRegistrationRequest', callback, registrationId, authorization, requestBody, forceRegistration);
+          register: (callback, request, authorization, requestBody) => {
```

Comment:
>authorization [](start = 40, length = 13)

I believe it's actually authentication. we're using this to prove our identity not determine our privileges imho.